src/components/Calendar.js

```diff
@@ -87,29 +121,51 @@ class Calendar extends React.Component {
     return weeks;
   };
 
-  _renderMonthTable = (styles) => {
+  _renderMonthTable = styles => {
     const weeks = this._getWeeks();
 
     return weeks.map(week => {
       return (
         <div key={week} style={styles.calendarWeek}>
           {week.map(day => {
-            const isCurrentMonth = day.isSame(moment.unix(this.state.currentDate), 'month');
-            const isSelectedDay = day.isSame(moment.unix(this.props.selectedDate), 'day');
+            const isCurrentMonth = day.isSame(
```

Comment:
is it possible to remove some of these assignments?

Reply:
I totally can - and I started to do this, but I actually think that the variable names make it clearer for the developer. For me when I read `isCurrentMonth` it's immediately obvious to me what the variable represents whereas parsing through
```
            day.isSame(
              moment.unix(this.state.currentDate),
              'month'
            );
```
is a little bit more difficult? what do you think